src/TicketTask.php

```diff
@@ -46,14 +46,22 @@ public static function getTypeName($nb = 0)
 
     public static function canCreate(): bool
     {
-        return (Session::haveRight(self::$rightname, parent::ADDALLITEM)
-              || Session::haveRight('ticket', Ticket::OWN));
+        return (Session::haveRightsOr(
+            self::$rightname,
+            [
+                parent::ADDALLITEM,
+                parent::ADDGROUPTICKET,
+                parent::ADDMYTICKET,
+                parent::ADD_AS_OBSERVER,
+                parent::ADD_AS_TECHNICIAN
+            ]
+        )
+        || Session::haveRight('ticket', Ticket::OWN));
```

Comment:
I don't know if we should keep this one?

Reply:
[a9e1b92](https://github.com/glpi-project/glpi/pull/17298/commits/a9e1b92a615d587dde8c4392f7fd0535c408b793)